cortx-motr.spec.in

```diff
@@ -71,10 +85,15 @@ BuildRequires:  make
 BuildRequires:  gcc
 BuildRequires:  gcc-c++
 %if %{rhel} <= 7
+%if  "%{?CONFIG_ARCH_TYPE}"=="x86_64"
 BuildRequires:  gccxml
 %else
+#BuildRequires:  castxml
+%endif
+%else
 BuildRequires:  castxml
 %endif
```

Comment:
Make it like this:

```
%if %{rhel} <= 7 && "%{?CONFIG_ARCH_TYPE}" == "x86_64"
BuildRequires:  gccxml
%else
BuildRequires:  castxml
%endif
```

Reply:
As castxml  is set up manually in the current build process and the same is not available in centos repository  for arm platform , The above code just marks the code for castxml package loading commented, So in order to avoid the existing x86 code from breaking. The above code is preferable for the time being till the castxml rpm is available form centos team